src/components/prescriptions/ImportGroupComponent.vue

```diff
@@ -1,6 +1,8 @@
 <template>
     <section class="prescriptionContainer">
-        <h3 class="prescriptionContainer__header">
+        <h3
+            class="prescriptionContainer__header"
+            tabindex="0">
             Zaimportowane recepty
             <span class="numOfPres">{{
```

Comment:
did you check whether SR reads num of elemt of list? If not youhave to implement it in span markup

Reply:
odczytywalo
przykladowo jak byly dwa to czytalo: zaimportowane recepty dwa

Reply:
ok so resolved